bindings/python/edlib.pyx

```diff
@@ -152,3 +154,91 @@ def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=
     cedlib.edlibFreeAlignResult(cresult)
 
     return result
+
+
+
+## function to output NICE alignments
+## only 'task=="path"' works
+
+def cigarToNiceAlignment(alignResult, query, target, gapSymbol="-"):
+    """ Output alignments from align() in NICE format
+    @param {dictionary} alignResult, output of align() 
+            Dictionary with following fields:
```

Comment:
I would not describe alignResult here again, it is enough that you mentioned it is output of align(), comments of align() can be observed to learn of its shape. We don't want to have to do changes on two places once we do changes to alignResult.

Reply:
I think that makes sense, removed